app/views/notify_mailer/notify_status_update.html.erb

```diff
@@ -1,6 +1,6 @@
 Dear <%= @resolver %>,<br>
 I would like to have a follow up on my recent ticket.<br>
-<%= @ticket_link %><br>
+<a href="<%= HOST_URL %>"><%= HOST_URL %></a>
```

Comment:
Why are we using HOST_URL constant, try getting base url from rails app, and use link_to or other erb alternatives rather than a href.

Reply:
The host_url constant is defined for email_links only.
reference https://github.com/joshsoftware/providesk_api/commit/62391a5494548a9fe8de209564c78b0dd2ea8990